AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/bookings-list.model.ts

```diff
@@ -105,4 +105,12 @@ export class BookingsDetailsModel {
     public get hasBookingConfirmationFailed(): boolean {
         return this.Status === 'Failed';
     }
+
+    public get hasBookingWithNoJudge(): boolean {
```

Comment:
Do any components still use this?

Reply:
it's being used in AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-status/booking-status.component.html for selecting the css class. I debated whether this method would best be place with that component. But I decided this is logic for the the model itself, and should go with the other status methods.

Reply:
Deleted my previous comment, you're right I'm not actually using it anywhere anymore. Ill remove it